src/main/java/games/strategy/triplea/ui/ObjectivePanel.java

```diff
@@ -613,30 +613,29 @@ public String getProperty(final String objectiveKey) {
   }
 
   public String getProperty(final String objectiveKey, final String defaultValue) {
-    return m_properties.getProperty(objectiveKey, defaultValue);
+    return properties.getProperty(objectiveKey, defaultValue);
   }
 
   public Set<Entry<Object, Object>> entrySet() {
-    return m_properties.entrySet();
+    return properties.entrySet();
   }
 }
 
 
 class ObjectivePanelDummyDelegateBridge implements IDelegateBridge {
```

Comment:
I believe that is correct. There are a few instances that delegates need do be 'dummied' for local operations like AI and battle calc.